site/source/conf.py

```diff
@@ -70,14 +70,15 @@
 TAGS = []
 smv_tag_whitelist = multiversion_regex_builder(TAGS)
 # Whitelist pattern for branches (set to None to ignore all branches)
-BRANCHES = ["0.2.x", "0.3.1.x", "0.3.2.x", "master"]
+BRANCHES = ["0.2.x", "0.3.1.x", "0.3.2.x", "0.3.4.x"]
```

Comment:
I think this needs 0.3.5.x

Reply:
I opted to rename the latest version to 0.3.5.x, so we can continue pushing to master and deploying from there if any changes are needed while still in the 0.3.5.x series. I think this is easier to maintain than having an additional 0.3.5.x branch. The flow would be to cut a branch when we move on to the next series (eg. for 0.3.4.x).

Reply:
I agree. I'm wondering why 0.3.5.x is not in the BRANCHES array